saleor/graphql/wishlist/schema.py

```diff
@@ -0,0 +1,24 @@
+import graphene
+
+from .mutations import (
+    WishlistAddProductMutation,
+    WishlistAddProductVariantMutation,
+    WishlistRemoveProductMutation,
+    WishlistRemoveProductVariantMutation,
+)
+
+
+# User's wishlist queries are located in the "saleor.graphql.account" module:
+#
+#     me {
+#         wishlist
+#     }
+class WishlistQueries(graphene.ObjectType):
```

Comment:
Do we need to store this class? I think that the comment contains enough info.

Reply:
I kept it in the case we want to add some general wishlist specific queries (e.g. from top of my head, how many likes does `product XYZ` has). Although if we will need it we can add it, so removing :+1: .